datahub-web-react/src/app/ingest/source/builder/constants.ts

```diff
@@ -147,6 +150,7 @@ export const PLATFORM_URN_TO_LOGO = {
     [BIGQUERY_URN]: bigqueryLogo,
     [CLICKHOUSE_URN]: clickhouseLogo,
     [COCKROACHDB_URN]: cockroachdbLogo,
+    [COUCHBASE_URN]: couchbaseLogo,
```

Comment:
in general, I prefer to have the UI forms get added in a separate PR. 

Small PRs are easier to review - and in this case, I like to have the UI changes reviewed by someone who does more frontend dev than I do